src/models/searchservice/response/directanswer/fieldvaluedirectanswers/UnknownDirectAnswer.ts

```diff
@@ -5,7 +5,7 @@ import { BaseFieldValueDirectAnswer } from '../BaseFieldValueDirectAnswer';
  *
  * @public
  */
-export interface UnknownFieldValueDirectAnswer<T = unknown> extends BaseFieldValueDirectAnswer<T>{
+export interface UnknownDirectAnswer<T = unknown> extends BaseFieldValueDirectAnswer<T>{
```

Comment:
This name we may need a bit more detail. We don't want someone to think this expresses any sort of unknown DA (snippets included). Maybe `UnknownFieldTypeDirectAnswer` or something like that?

Reply:
I see, here I think I prefer FieldValue over FieldType as a qualifier since this doesn't apply to FeaturedSnippets

Reply:
agree, I think keeping UnknownFieldValueDirectAnswer is good